README.md

```diff
@@ -28,14 +11,12 @@ This is a [Devise](https://github.com/plataformatec/devise) extension to add [Tw
     * [Manually](#manually)
     * [Final steps](#final-steps)
 * [Custom Views](#custom-views)
-  * [Request a phone call](#request-a-phone-call)
```

Comment:
Request a phone call was readded by https://github.com/MidwestAccessCoalition/twilio-verify-devise/pull/97 so it should be kept.